### DISCUSSION TITLE
Modified by the original open source agreement Apache license v2.0?

### DISCUSSION THREAD

**Discussion by ytc301** [minio/minio#12895] [Q&A] [answered]
Does the current open source protocol agpl V3 affect users of the original Apache license v2.0  ？

AGPL V3 requires users to open source their own applications. If there is any impact, we will consider changing other storage schemes

**Comment by harshavardhana** [comment, marked as answer]
https://min.io/compliance please read the compliance page. Please consult your legal department or a lawyer to ensure that your usage of our software is in compliance with the FOSS license terms.
